docs/web-console-guide_5/master.adoc

```diff
@@ -90,7 +153,35 @@ include::topics_5/web-report-static.adoc[leveloffset=+2]
 ////
 
 // Using Custom Rules
-include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+//include::topics_5/web-custom-rules.adoc[leveloffset=+1]
+
+== Using global rules and labels
+
+Global rules and labels apply to all projects and are automatically applied to analyses for which they fit the definitions of a project, unless they are explicitly excluded in the *Advanced options* of the project. The {ProductShortName} includes a preconfigured set of global rules and labels and you can define your own custom global rule and labels.
+
+These rules and labels are displayed and defined on the *Rules configuration* and the *Labels configuration* pages, respectively.
+
+=== Using global rules
```

Comment:
Make this into a level 2 heading (get rid of the "Using rules/labels" level 2 heading above it).

Reframe this. It's not about using global rules. It's about creating custom rules. 

This applies to labels as well.